File: Domain/Bus.cs
namespace Model
{
    public class Bus
    {
        public Guid? BusServiceId { get; set; }

        public string? RegistrationNumber { get; set; }

        public bool IsAvailable { get; set; }

        public List<Seat>? Seats { get; set; }

    }
}

File: ReservedYouFrontEnd/src/app/models/Bus.ts
export class Bus {
    registrationNumber:string;
    busServiceId:string;
    isAvailable:string;

    constructor(rNo:string,bId:string,isAvail:string) {
        this.registrationNumber = rNo;
        this.busServiceId = bId;
        this.isAvailable = isAvail;
    }
}